tests/unit/igraph_triangle_lattice.c

```diff
@@ -0,0 +1,82 @@
+/*
+   IGraph library.
+   Copyright (C) 2021  The igraph development team <igraph@igraph.org>
```

Comment:
```suggestion
   Copyright (C) 2022  The igraph development team <igraph@igraph.org>
```

Reply:
When a review commit is addressed, please click the "Resolve conversation" button, so we know it's done. I'll do so for all of them now.